unimus-backup-exporter.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+#This is a Unimus to Git API to export your backups to your Git Repo
+
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
```

Comment:
cleanup, all code either in functions, or at the bottom before the `main` call

Reply:
Code has been reorganized.